From 6b6cd4acc9243cad8efd3edd98f3fce1b258f815 Mon Sep 17 00:00:00 2001
From: ChenXiaoSong <chenxiaosong@kylinos.cn>
Date: Sun, 7 Dec 2025 11:22:38 +0800
Subject: [PATCH 19/30] smb/client: remove useless elements from nt_errs array

NT_STATUS_OK are zero, it is not needed.

The last element in the array is no longer needed.

Signed-off-by: ChenXiaoSong <chenxiaosong@kylinos.cn>
---
 fs/smb/client/nterr.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/fs/smb/client/nterr.c b/fs/smb/client/nterr.c
index 8d0007db6af9..faacec11c611 100644
--- a/fs/smb/client/nterr.c
+++ b/fs/smb/client/nterr.c
@@ -12,7 +12,6 @@
 #include "nterr.h"
 
 struct nt_err_code_struct nt_errs[] = {
-	{"NT_STATUS_OK", NT_STATUS_OK},
 	{"NT_STATUS_PENDING", NT_STATUS_PENDING},
 	{"NT_STATUS_MEDIA_CHANGED", NT_STATUS_MEDIA_CHANGED},
 	{"NT_STATUS_END_OF_MEDIA", NT_STATUS_END_OF_MEDIA},
@@ -684,7 +683,6 @@ struct nt_err_code_struct nt_errs[] = {
 	{"NT_STATUS_NO_SUCH_JOB", NT_STATUS_NO_SUCH_JOB},
 	{"NT_STATUS_NO_PREAUTH_INTEGRITY_HASH_OVERLAP",
 	 NT_STATUS_NO_PREAUTH_INTEGRITY_HASH_OVERLAP},
-	{NULL, 0}
 };
 
 unsigned int nt_err_num = sizeof(nt_errs) /
-- 
2.43.0

